### PR TITLE
Remove `prevOnRamp` from multi onramp

### PR DESCRIPTION
**Pull request by RayXpub**
## Motivation

Missed in https://github.com/smartcontractkit/ccip/pull/985.

With the introduction of the `NonceManager` the `prevOnRamp` sequencing check isn't performed in the onramp anymore.

## Solution

Remove `DestChainConfig.prevOnRamp` and `DestChainConfigArgs.prevOnRamp`.

### CODE REVIEW

File: contracts/src/v0.8/ccip/onRamp/EVM2EVMMultiOnRamp.sol [hunk 1]
```diff
@@ -146,8 +146,7 @@ contract EVM2EVMMultiOnRamp is IEVM2AnyOnRampClient, ITypeAndVersion, OwnerIsCre
   /// @dev Struct to hold the configs for a destination chain
   struct DestChainConfig {
     DestChainDynamicConfig dynamicConfig; // ──╮ Dynamic configs for a destination chain
```
Comment (elatoskinas) [inline contracts/src/v0.8/ccip/onRamp/EVM2EVMMultiOnRamp.sol]: nit: should we remove the struct packing line for the dynamicConfig struct?

File: contracts/src/v0.8/ccip/test/onRamp/EVM2EVMMultiOnRamp.t.sol [hunk 2]
```diff
@@ -197,16 +195,13 @@ contract EVM2EVMMultiOnRamp_applyDestChainConfigUpdates is EVM2EVMMultiOnRampSet
     );
     bool isNewChain = true;
 
-    if (destChainConfigArgs.destChainSelector == DEST_CHAIN_SELECTOR) {
-      destChainConfigArgs.prevOnRamp = address(0);
-      isNewChain = false;
-    }
+    if (destChainConfigArgs.destChainSelector == DEST_CHAIN_SELECTOR) isNewChain = false;
```
Comment (elatoskinas) [inline contracts/src/v0.8/ccip/test/onRamp/EVM2EVMMultiOnRamp.t.sol]: suggestion: L196 and L198 can be collapsed to:

```
bool isNewChain = destChainConfigArgs.destChainSelector != DEST_CHAIN_SELECTOR;
```

**Review by elatoskinas — APPROVED** [review]

File: contracts/src/v0.8/ccip/onRamp/EVM2EVMMultiOnRamp.sol (hunk 1 above)
Comment (RayXpub) [reply contracts/src/v0.8/ccip/onRamp/EVM2EVMMultiOnRamp.sol]: Good catch, updated.

File: contracts/src/v0.8/ccip/test/onRamp/EVM2EVMMultiOnRamp.t.sol (hunk 2 above)
Comment (RayXpub) [reply contracts/src/v0.8/ccip/test/onRamp/EVM2EVMMultiOnRamp.t.sol]: Updated

**Review by connorwstein — APPROVED** [review]

**Pull request merged**